webpack/assets/javascripts/react_app/components/common/DocumentationLink/index.js

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { MenuItem } from 'patternfly-react';
-import Icon from '../Icon';
```

Comment:
Can we remove the Icon component itself?

Reply:
I removed the Icon folder. What else should be remove?